README.md

```diff
@@ -1,29 +1,18 @@
-# Mantis Faucet Web
+# SC_EVM Faucet
 
-Web interface for Mantis Faucet
 
-## Prerequisites
+## 1. Notice
 
-- [Node.js](https://nodejs.org/en/) - tested with v12
-  - (optional) [nvm](https://github.com/nvm-sh/nvm) - you can use nvm to automatically pick-up the Node version you need
-- [yarn](https://classic.yarnpkg.com/en/) - tested with v1.22.5
+As of [insert date], IOG will no longer be updating or maintaining this repo.
 
-## Initialize the project
+After three months of experimentation with the proof-of-concept EVM sidechain testnet, we have gathered valuable learnings from this experience and this innovative playground. New use cases and functionality were tested, feedback from the community was gathered, and we are now shifting our focus back to the strategic evolution of our sidechains approach.  
+  
+All information included below is considered publicly available and is free to be leveraged by developers to fork it to build and experiment with their own EVM sidechain solution. Should you have any questions, please reach out to our team on our IOG Discord server.  
+## 2. Description
```

Comment:
nit: no newline around here. Sometimes, markdown is interpreted differently with or without newline